### ISSUE TITLE
Dynamic Warning Updates

### ISSUE THREAD

**Issue by mammuth** [Blueshoe/djangocms-layouter#2]
Dragging and Dropping in the structure view does not yet update the warnings within the
plugin.

**Comment by SteinRobert** [comment]
Exactly. I've tried to dig into the JS, but I did not find a way to hook into the corresponding event. It seems like Django-CMS cancels any propagation, which make it pretty much impossible to catch that event.
